src/pages/settings/Profile/PersonalDetails/AddressPage.js

```diff
@@ -122,7 +122,7 @@ function AddressPage({privatePersonalDetails, route}) {
             <HeaderWithBackButton
                 title={translate('privatePersonalDetails.address')}
                 shouldShowBackButton
-                onBackButtonPress={() => Navigation.goBack(ROUTES.SETTINGS_PERSONAL_DETAILS)}
+                onBackButtonPress={() => Navigation.goBack(ROUTES.SETTINGS_PROFILE)}
```

Comment:
I think it's find to skip param. Simply call `Navigation.goBack()`
This is consistent with DisplayName page

I also tested this flow:
- deep link to https://dev.new.expensify.com:8082/settings/profile/address
- Press Back button
- It correctly goes to Profile page

Reply:
Agree it can be skipped. Profile page is now in central pane so it's always mounted under the RHP and we don't need to pass it as param for go back.

Reply:
Yes, fixed in all places